ui/src/config/section/infra/hosts.js

```diff
@@ -24,6 +24,10 @@ export default {
   icon: 'desktop-outlined',
   permission: ['listHostsMetrics'],
   resourceType: 'Host',
+  filters: () => {
+    const filters = ['enabled', 'disabled']
```

Comment:
@kishankavala could you explore other states for the resources. For example as an admin I only want to see hosts in Alert, Maintenance and other useful states? Pl do the same for other resources (pods, cluster) in this PR

Reply:
@rohityadavcloud cc @kishankavala I guess this PR adds filter for host `resourcestate` which can only be,
```
Creating,
Enabled,
Disabled,
ErrorInPrepareForMaintenance,
PrepareForMaintenance,
ErrorInMaintenance,
Maintenance,
Error,
Degraded
```

Not sure if we have filtering for host state as well which can be host `status` as well which can be,

```
Creating
Connecting
Up
Down
Disconnected
Alert
Removed
Error
Rebalancing
Unknown
```

Reply:
@kishankavala any update on this? Thanks.